doc/docs/proxy_settings.md

```diff
@@ -0,0 +1,40 @@
+# Starting nRF Connect for Desktop with proxy settings
+
+You can run nRF Connect for Desktop with custom proxy network settings if your network or OS policy requires them to access the application.
+
+nRF Connect for Desktop has the same proxy support as [Google Chromium](https://www.chromium.org/developers/design-documents/network-settings),
+and should in most cases pick up the proxy settings configured in the underlying OS. However, in some cases you may need to add some extra configuration, as
+described below.
+
+## Available proxy settings
+
+You can run nRF Connect for Desktop with the following proxy setting configuration:
+
+| Proxy setting flag  | Description | Parameters  |
+|---------------------|-------------|-------------|
+| `--no-proxy-server` | Do not use proxy. This overrides any other proxy settings provided. | None  |
+| `--proxy-server`    | Use a custom proxy configuration. | - `<scheme>=<uri>[:<port>][;...]` - A semi-colon-separated mapping of list scheme to URL/port pairs. </br> - `<uri>[:<port>]` - A single URI with optional port to use for all URLs. </br> - `"direct://"` - The special value that causes all connections to not use a proxy. |
```

Comment:
With #995 we slightly limited what users can use as parameters here. The `<scheme>=` and `direct://` are now only partially supported, and very few people will have a need for them, so I suggest to not mention them here.
```suggestion
| `--proxy-server`    | Use a custom proxy configuration. | `<uri>[:<port>]` - A single URI with optional port to use for all URLs. |
```